### DISCUSSION TITLE
如何在本地调试？

### DISCUSSION THREAD

**Discussion by aozaki-kuro** [spencerwooo/onedrive-vercel-index#123] [Q&A] [answered]
鉴于想自行修改的样式有点多，想要尽量在本地把样式调整完后再 commit 上来…
`npm install` 后 `yarn dev` 可以跑起来但是拉取不到内容，应该是没有添加相应 token 的原因。
不过我是小白一枚，搜索半天完全也不太清楚应该如何在本地添加 `ACCESS_TOKEN` 等token信息😂
希望有大佬可以协助解答一下

**Comment by spencerwooo** [comment, marked as answer]
在本地目录下新建文件名为 `.env`（或 `.env.local`）的纯文本文件（没有其他后缀，就叫这个名字），然后写入类似格式的内容即可：

```
CLIENT_SECRET=xxx
ACCESS_TOKEN=xxx
REFRESH_TOKEN=xxx
```
